docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-mobile-data-model/index.md

```diff
@@ -1,14 +1,25 @@
 ---
 title: "Mobile"
-sidebar_position: 200
+sidebar_position: 800
 hide_title: true
 ---
 
 ```mdx-code-block
 import Badges from '@site/src/components/Badges';
 import ThemedImage from '@theme/ThemedImage';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+import {versions} from '@site/src/componentVersions';
 ```
-<Badges badgeType="dbt-package Release" pkg="mobile"></Badges>
+<Badges badgeType="dbt-package Release" pkg="mobile"></Badges>&nbsp;
+<Badges badgeType="Maintained"></Badges>&nbsp;
+<Badges badgeType="SCL"></Badges>
+
+:::info
+
+This package has been superseded by the [Digital Unified](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/index.md) that combines data from both web and mobile sources. For more information, see the Digital Unified page.
```

Comment:
Why do we call it 'Digital Unified' here? It should be Unified Digital.

Reply:
Yeah that's just me getting it wrong